src/components/Sum.jsx

```diff
@@ -1 +1,22 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum(props = 0) {
+  const aNum = +props.a || 0;
+  const bNum = +props.b || 0;
```

Comment:
props object is always there
```suggestion
export function Sum({ a = 0, b = 0 }) {
  const aNum = +props.a;
  const bNum = +props.b;
```